src/components/main/flight/CheckoutWithFlightScreen.js

```diff
@@ -142,254 +93,186 @@ export default class CheckoutWithFlightScreen extends React.Component {
+            <View>
+              <Text style={styles.paragraph}>
+                {i18n.t("Flight Number")} {flightChars} {flightNums}
               </Text>
+              <Text style={styles.emissions}>{footprint}</Text>
+              <Text style={styles.emissionsUnits}>{i18n.t("tons of CO2")}</Text>
             </View>
-          </View>
-          <View style={styles.iterateGroup}>
-            {/*Subtract tree*/}
-            <TouchableOpacity
-              style={[
-                styles.iterators,
-                {
-                  backgroundColor:
-                    this.state.treeNum == 0 ? COLORS.grey : COLORS.lightgreen,
-                },
-              ]}
-              onPress={() => this.handleRemove()}>
-              <Feather
-                style={[
-                  styles.iteratorIcon,
-                  {
-                    color:
-                      this.state.treeNum == 0 ? COLORS.white : COLORS.lightblue,
-                  },
-                ]}
-                name="minus"
-              />
-            </TouchableOpacity>
-            {/*Tree counter*/}
-            <View style={[styles.treeCounter, { backgroundColor: color }]}>
-              <Text style={styles.treeCountText}>{treeNum}</Text>
+
+            {/* Number of Trees */}
+            <View style={styles.treeNumRow}>
+              <View style={styles.treeNumCol}>
+                <Image source={tree} style={styles.treeImg} />
+              </View>
+              <View style={styles.treeNumCol}>
+                <Text style={styles.numTrees}>{treeNum}</Text>
+              </View>
             </View>
-            {/*Add tree*/}
-            <TouchableOpacity
-              style={[styles.iterators, { backgroundColor: COLORS.lightgreen }]}
-              onPress={() => this.handleAdd()}>
-              <Feather
-                style={[styles.iteratorIcon, { color: COLORS.lightblue }]}
-                name="plus"
-              />
-            </TouchableOpacity>
-          </View>
-          <View style={styles.bottomText}>
-            {/*Years to neutralize carbon footprint*/}
-            <Text style={styles.midBlueText}>
-              {i18n.t("YEARS TO COMPENSATE")}
-            </Text>
-            {years != Infinity && (
-              <Text style={styles.bigBlueText}>{years}</Text>
-            )}
-            {years == Infinity && (
-              <Text style={styles.bigBlueText}>&#x2014;</Text>
-            )}
-          </View>
-          <Dash
-            style={styles.dashedLine}
-            dashColor={COLORS.lightgrey}
-            dashGap={0}
-          />
-          <View style={styles.receiptContainer}>
-            <View style={styles.textRow}>
-              {/*Total trees donated in transaction*/}
-              <Text style={styles.receiptTextLeft}>
-                {i18n.t("TOTAL TREES")}
+
+            {/* Add or Remove Trees */}
+            <View style={styles.addRemoveView}>
+              {/* Minus Icon */}
```

Comment:
For this addRemove View, can you reduce the margin above it just a bit? So it's centered between the tree and the text?